app/build.gradle

```diff
@@ -20,7 +20,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.2.0'
```

Comment:
Please revert the gradle version

Reply:
Noted, I have reverted to the previous version, and the local build works fine